Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -117,11 +117,18 @@ class MessageDTO: NSManagedObject {
             .init(format: "createdAt > channel.truncatedAt")
         ])
 
+        // Some pinned messages might be in the local database, but should not be fetched
+        // if they are not in the current batch of messages.
```

Comment:
Same here - the "current batch of messages" is imo a little bit confusing wording.